website/public/locales/hu/labelling.json

```diff
@@ -1,24 +1,79 @@
+  "fails_task.one_desc": "Nem követi az utasítást vagy feladatot",
+  "fails_task.question": "Ez hibás válasz a kért feladathoz?",
+  "follows_instructions": "Követi az utasításokat",
+  "harmful": "Ártalmas",
+  "harmful.one_desc.line_1": "A tartalom indokolatlan mértékben ártalmas ebben a környezetben.",
+  "harmful.one_desc.line_2": "Az ártalon fizikai vagy mentális kárt vagy sérülést jelent. Az indokolatlan mérték egy szintkorlátot jelent, például a bőrsérülés nem túlzás egy orvosi beavatkozás esetében.",
+  "harmless": "Ártalmatlan",
+  "hate_speech": "Gyűlöletbeszéd",
+  "hate_speech.explanation": "A tartalom sértő vagy fenyegető, és valamely védett tulajdonsággal szembeni előítéletet fejez ki. Az előítéletesség itt nem ésszerű alapokon nyugvó előítéletet jelent. A védett tulajdonságok közé tartozik a nem, az etnikai hovatartozás, a vallás, a szexuális irányultság és a hasonló jellemzők.",
+  "hateful": "Gyűlölködő",
+  "hateful.one_desc.line_1": "A tartalom sértő vagy fenyegető, és valamely védett tulajdonsággal szembeni előítéletet fejez ki",
+  "hateful.one_desc.line_2": "Az előítéletesség itt nem ésszerű alapokon nyugvó előítéletet jelent. A védett tulajdonságok közé tartozik a nem, az etnikai hovatartozás, a vallás, a szexuális irányultság és a hasonló jellemzők.",
+  "helpful": "Hasznos",
+  "helpful.one_desc": "Magas minőségben végzi el a feladatot",
+  "high_quality": "Magas minőségű",
+  "humorous": "Humoros",
+  "humorous.one_desc": "Humoros tartalom, beleértve a szarkazmust is",
+  "inappropriate": "Nem megfelelő",
+  "inappropriate.one_desc": "Egy személyi asszisztens számára nem helyénvaló",
+  "judgemental": "Ítélkező",
+  "judgemental.one_desc": "Értékítéletet tartalmaz",
+  "label_highlighted_flag_instruction": "Válaszd ki az összeset, amely a kiemelt üzenetre érvényes:",
+  "label_highlighted_likert_instruction": "A kiemelt üzenet értékelése:",
+  "label_highlighted_yes_no_instruction": "Válaszold meg a kiemelt üzenetekkel kapcsolatos kérdéseket:",
+  "label_message_flag_instruction": "Válaszd ki az összeset, amely az üzenetre érvényes:",
+  "label_message_likert_instruction": "A válasz értékelése:",
+  "label_message_yes_no_instruction": "Válaszold meg az üzenettel kapcsolatos kérdéseket:",
+  "lang_mismatch": "Nem {{language}}",
+  "lang_mismatch.explanation": "Nem {{language}} nyelven van.",
+  "low_quality": "Alacsony minőségű",
+  "misleading": "Félrevezető",
+  "misleading.one_desc": "Helytelen vagy félrevezető szöveget tartalmaz",
+  "moral_judgement": "Erkölcsileg ítélkező",
+  "moral_judgement.explanation": "Erkölcsi ítéletet fogalmaz meg.",
+  "non_judgemental": "Nem ítélkező",
+  "non_sexual": "Nem szexuális",
+  "not_spam": "Nem kéretlen",
+  "not_spam.explanation": "Megfelelő az Open Assistant betanításához.",
+  "not_appropriate": "Nem megfelelő",
+  "not_appropriate.explanation": "Egy személyi asszisztens számára nem helyénvaló.",
+  "ordinary": "Közönséges",
+  "pii": "PII-t tartalmaz",
```

Comment:
ezt kérlek egy magyarul is érthetőre, hátha nem látszik a leírás (anno Személyes Adat volt)

Reply:
Átírtam